src/utils/crypto/libsodium.ts

```diff
@@ -290,6 +291,17 @@ export async function deriveSensitiveKey(passphrase: string, salt: string) {
     }
 }
 
+export async function deriveInteractiveKey(passphrase: string, salt: string) {
+    // default algo can change in future when we upgrade library.
+    // this assert act as a safegaurd for us to identify any such issue.
+    assert(
```

Comment:
we would need this check at more places... if this is a concern

Reply:
Agree. We can just do it once during the app start. Any suggestions on where should I put this check?

Reply:
we should put this check  in  sensitive key too